src/main/webapp/app/shared/select/CancerTypeSelect.tsx

```diff
@@ -8,11 +8,12 @@ import { IRootStore } from 'app/stores/createStore';
 import { connect } from '../util/typed-inject';
 import { ICancerType } from '../model/cancer-type.model';
 import { ITEMS_PER_PAGE } from '../util/pagination.constants';
+import { getCancerTypeName } from 'app/shared/util/utils';
 
 interface ICancerTypeSelectProps extends SelectProps, StoreProps {}
 
 const getAllMainTypes = (cancerTypeList: ICancerType[]) => {
-  return _.uniq(cancerTypeList.filter(cancerType => cancerType.level >= 0)).sort();
+  return _.uniq(cancerTypeList.filter(cancerType => cancerType.level <= 0)).sort();
```

Comment:
Why is this filtering by checking if levels <= 0? Wouldn't this remove all of the cancer types?

Reply:
@scarrero4660 within the list of cancer types, we have concepts `main type` and `subtype`. They are the concepts from OncoTree. They are also called `Cancer Type` and `Cancer Type Details`. Any level 0 are main type, and any level >0 are subtype. OncoKB also adds additional cancer types for our purpose, such as `All Solid Tumors`, for these, we use level -1.